CHANGELOG.md

```diff
@@ -1,3 +1,269 @@
+* [#574:](https://github.com/pysal/spaghetti/issues/574) [JOSS review] Manuscript summary 
+* [#608:](https://github.com/pysal/spaghetti/pull/608) update JOSS paper cite and figure size 
+* [#607:](https://github.com/pysal/spaghetti/pull/607) update JOSS paper text 
+* [#605:](https://github.com/pysal/spaghetti/pull/605) [WIP] Add paper references, etc. 
+* [#586:](https://github.com/pysal/spaghetti/issues/586) additional functions & methods from tutorials 
+* [#606:](https://github.com/pysal/spaghetti/pull/606) improve point snap / Moran figure for JOSS 
+* [#588:](https://github.com/pysal/spaghetti/pull/588) addressing issue #576 
+* [#603:](https://github.com/pysal/spaghetti/pull/603) add to quick start notebook 
+* [#592:](https://github.com/pysal/spaghetti/pull/592) addressing #582: update spaghetti.Network docstring 
+* [#602:](https://github.com/pysal/spaghetti/pull/602) Bump actions/cache from v2 to v2.1.4 
+* [#601:](https://github.com/pysal/spaghetti/pull/601) update docs only on release 
+
+
+<a name="mgwr"></a>
+### mgwr
+
+
+<a name="spglm"></a>
+### spglm
+
+
+<a name="spint"></a>
+### spint
+
+
+<a name="spreg"></a>
+### spreg
+* [#76:](https://github.com/pysal/spreg/pull/76) Update conf and rerelease 
+* [#75:](https://github.com/pysal/spreg/pull/75) Update GHA & CI 
+* [#59:](https://github.com/pysal/spreg/issues/59) Add Python 3.9 testing 
+* [#74:](https://github.com/pysal/spreg/pull/74) Fixing bug with order of variables' names changing randomly in regimes. 
+* [#66:](https://github.com/pysal/spreg/pull/66) Bump actions/setup-python from 2 to 2.2.2 
+* [#67:](https://github.com/pysal/spreg/pull/67) Bump actions/cache from 2 to 2.1.5 
+* [#68:](https://github.com/pysal/spreg/pull/68) Bump codecov/codecov-action from 1 to 1.5.0 
+* [#69:](https://github.com/pysal/spreg/pull/69) Bump actions/upload-release-asset from 1 to 1.0.2 
+* [#70:](https://github.com/pysal/spreg/pull/70) Bump actions/create-release from 1 to 1.1.4 
+* [#65:](https://github.com/pysal/spreg/pull/65) Bump actions/cache from v2 to v2.1.4 
+
+
+<a name="spvcm"></a>
+### spvcm
+
+
+<a name="tobler"></a>
+### tobler
+* [#144:](https://github.com/pysal/tobler/pull/144) add pycno to docs 
+* [#143:](https://github.com/pysal/tobler/pull/143) remove css from docs conf 
+* [#142:](https://github.com/pysal/tobler/pull/142) bump version 0.8 
+* [#141:](https://github.com/pysal/tobler/pull/141) add pycnophylactic interpolation 
+* [#95:](https://github.com/pysal/tobler/issues/95) Add Pycnophylactic Interpolation 
+* [#79:](https://github.com/pysal/tobler/issues/79) sentinel data 
+* [#135:](https://github.com/pysal/tobler/pull/135) ENH: support categorical variables in area_interpolate 
+* [#96:](https://github.com/pysal/tobler/issues/96) vector data as a dasymetric mask 
+* [#132:](https://github.com/pysal/tobler/pull/132) bump version 
+
+
+<a name="mapclassify"></a>
+### mapclassify
```

Comment:
@sjsrey do you want to cut a release of mapclassify with https://github.com/pysal/mapclassify/pull/110 and https://github.com/pysal/mapclassify/pull/114?

Reply:
will do so tonight.